src/ansys/simai/core/data/models.py

```diff
@@ -38,6 +114,43 @@ class ModelConfiguration:
     simulation_volume: Dict[str, Any] = None
     project_id: str = None
 
+    @property
+    def domain_of_analysis(self) -> DomainOfAnalysis | None:
+        """The Domain of Analysis of the model configuration."""
+        if not self.simulation_volume:
+            return None
+
+        doa = DomainOfAnalysis()
+
+        doa.Length = self._get_doa_axis("X")
+        doa.Width = self._get_doa_axis("Y")
+        doa.Height = self._get_doa_axis("Z")
+
+        return doa
+
+    @domain_of_analysis.setter
```

Comment:
A docstring is not required for a setter following a property as the setter only exposes set and get methods.
Examples can be included though.